src/main/resources/db/changelog/002_Add_table_users.sql

```diff
@@ -0,0 +1,11 @@
+-- liquibase formatted sql
+-- changeset author:oskachkov
+
+CREATE TABLE users(
```

Comment:
In Postgres, attempting to create a table that already exists will result in an error stating that the "relation already exists". To avoid such errors, the IF NOT EXISTS clause can be used with the CREATE TABLE command.

Reply:
I used the liquidbase instead